src/lib/radio/radio.spec.ts

```diff
@@ -274,6 +274,29 @@ describe('MdRadio', () => {
       expect(groupInstance.value).toBe('water');
       expect(changeSpy).not.toHaveBeenCalled();
     });
+
+    it('should update checked status if changed value to radio group\'s value', () => {
```

Comment:
Instead of escaping the `'`, you can use a different type of quote for the string (usually backtick)